src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -848,15 +861,25 @@ Here's an example of a SCEP provisioner in `$(step path)/config/ca.json`:
     "type": "SCEP",
     "name": "scepca",
     "forceCN": true,
-    "challenge": "secret1234"
+    "challenge": "secret1234",
+    "minimumPublicKeyLength": 2048,
+    "includeRoot": true,
+    "encryptionAlgorithmIdentifier": 2,
 }
 ```
 
-- The `forceCN` parameter is optional. It behaves the same as `forceCN` in the ACME provisioner, and it defaults to false.
+- The `forceCN` parameter is optional. 
+It behaves the same as `forceCN` in the ACME provisioner, and it defaults to false.
+- `challenge` is the secret shared between the provisioner and SCEP clients. By default no secret is used.
+- The `minimumPublicKeyLength` parameter can be used to set the minimum length of public keys submitted by a client. Defaults to 2048.
+- When `includeRoot` is set to true, the root CA certificate will be returned in `GetCACert` requests in addition to the intermediate CA certificate. Defaults to false.
```

Comment:
Was there a particular client that motivated this option?

Reply:
The macOS client. There's some notes about this in https://github.com/smallstep/certificates/issues/746. 

It's been quite a while since then, but I remember there were two cases. One of them required the root to be included in addition to the intermediate, otherwise the request would fail. The other case would happily continue without. They were related to whether the macOS configuration profile had the chain included or not, to function as a trust anchor. So far I don't know about other clients that required this. The macOS client may or may not be doing the right thing, because SCEP's known to have some interoperability issues related to trust anchors. At least this configuration option allows both cases to work.

If we start the implementation of the changes discussed internally, that might affect the macOS client and/or this setting too. I'll have to remain aware of this to ensure compatibility.

Reply:
Since it's such a specific setting, seems like it'd be worth mentioning that it's a useful for macOS SCEP clients. Could even link to the issue (I've done that in a couple other places in the docs).